Microsoft.HealthVault.Fhir/Transformers/ThingBaseToFhir.cs

```diff
@@ -22,88 +23,65 @@ namespace Microsoft.HealthVault.Fhir.Transformers
     public static partial class ThingBaseToFhir
     {
         /// <summary>
-        /// Transforms a HealthVault thing into a FHIR Observation
+        /// Transforms a HealthVault thing into a FHIR Resource
         /// </summary>
         /// <param name="thing">The HealthVault thing to transform</param>
-        /// <returns>A FHIR observation based on the HealthVault thing</returns>
-        public static Observation ToFhir(this ThingBase thing)
+        /// <returns>A FHIR resource based on the HealthVault thing</returns>
+        public static Resource ToFhir(this ThingBase thing)
```

Comment:
>Resource ToFhir(this ThingBase thing) [](start = 22, length = 37)

We should probably remove this and use the ToFhir method that uses generics below as the ToFhir function now. We will need to transform from ThingBase to any Resource